src/en/spyfakku/src/eu/kanade/tachiyomi/extension/en/spyfakku/Filters.kt

```diff
@@ -0,0 +1,40 @@
+package eu.kanade.tachiyomi.extension.en.spyfakku
+
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+
+fun getFilters(): FilterList {
+    return FilterList(
+        SortFilter("Sort by", getSortsList),
+        OrderFilter("Order by", getOrdersList),
```

Comment:
there is a `SortFilter` abstract clas which holds sort and ascending/descending state, use that to combine sort and order